logging/opensearch/securityconfig/config.yml

```diff
@@ -65,9 +65,11 @@ config:
     # Set filtered_alias_mode to 'nowarn' to allow more than 2 filtered aliases per index silently
     #filtered_alias_mode: warn
     do_not_fail_on_forbidden: true
-    #kibana:
-    # Kibana multitenancy
-    #multitenancy_enabled: true
+    kibana:
+    # Kibana multitenancy 
+      multitenancy_enabled: true
```

Comment:
I know this section was enabled to disable private tenant but does multitenancy need to be enabled as well?

Reply:
This flag relates to an OpenSearch Dashboards (OSD) concept of tenancy and NOT to the SAS Viya concept of tenancy.  So, we are disabling a feature in OSD that gave every user a private area for their own content while also enabling the overall OSD features that allow tenant-specific content to be isolated from other tenants.  The OSD 'private' tenant space could be a nice feature but given its current implementation in OSD, it more likely to lead to confusion rather than be helpful.